Example/Podfile.lock

```diff
@@ -1,10 +1,10 @@
 PODS:
-  - Expecta (1.0.6)
+  - Expecta (1.0.5)
```

Comment:
Why downgrade Expecta and other testing libraries?

Reply:
Didn't downgrade by purpose. Will investigate.